zephyr/src/net/spotflow_mqtt.c

```diff
@@ -64,7 +64,7 @@ static uint8_t tx_buffer[APP_MQTT_BUFFER_SIZE];
 int spotflow_mqtt_poll()
 {
 	/* 1) Network I/O: wait up to 1 000 ms for socket readability */
-	int rc = zsock_poll(mqtt_client_toolset.fds, 1, 1000);
+	int rc = zsock_poll(mqtt_client_toolset.fds, 1, 10);
```

Comment:
```suggestion
	/* 1) Network I/O: wait up to 10 ms for socket readability */
	int rc = zsock_poll(mqtt_client_toolset.fds, 1, 10);
```

Supposing it was intentional.

Reply:
yes